### DISCUSSION TITLE
Remove duplicate Settings?

### DISCUSSION THREAD

**Discussion by Varorbc** [xunit/xunit#2919] [General]
Because a project that references the xunit package will necessarily reference the Microsoft.NET.Test.Sdk package

https://github.com/xunit/xunit/blob/47644d5438ecdcc54deb8998a34c305b1912f96d/src/xunit.core/build/xunit.core.props#L4-L11

https://github.com/microsoft/vstest/blob/65d9344b73c9cec20b4f42fc27e6d9ace5ca3c1c/src/package/Microsoft.NET.Test.Sdk/Microsoft.NET.Test.Sdk.props

**Comment by bradwilson** [comment]
> Because a project that references the xunit package will necessarily reference the Microsoft.NET.Test.Sdk package

This is not strictly speaking true. The dependency for `Microsoft.NET.Test.SDK` is only needed if the developer is also including `xunit.runner.visualstudio`, which is an optional dependency.

**Comment by Varorbc** [comment]
I found that things like nunit,mstest, don't have that. The problem I have now is that I have a library package that references xunit, which becomes a test project, and I just want it to be a library project.

**Reply by bradwilson** [reply]
If you're linking against xUnit.net to extend it, then you want to link against one of the extensibility packages instead (`xunit.extensibility.core` for `xunit.core.dll` and/or `xunit.extensibility.execution` for `xunit.execution.*.dll`). Linking against the `xunit.core` package (directly, or via the `xunit` package) is only for test projects.
